paper-server/patches/sources/net/minecraft/world/level/GameRules.java.patch

```diff
@@ -243,6 +247,14 @@
              GameRules.VisitorCaller<T> visitorCaller,
              Class<T> valueClass,
              FeatureFlagSet requiredFeatures
+@@ -608,6 +_,7 @@
+             this.visitorCaller.call(visitor, key, this);
+         }
+ 
++        @Override
```

Comment:
Paper comments :+1:

Reply:
Since the method is already here, maybe it would be better to just not include the optional Override annotation to reduce diff?

Reply:
Yea, probably a good call.